src/components/connection-method-selection-dialog.tsx

```diff
@@ -155,12 +155,10 @@ export const ConnectionMethodSelectionDialog: FC<ConnectionMethodSelectionDialog
         }
         return method;
       });
-    const hasInstalledWallet = filteredMethods.some(
-      method => method.isInstalled
-    );
     const getTier = (method: LikeCoinWalletConnectorMethod) => {
-      if (hasInstalledWallet) {
-        return method.isInstalled ? 1 : 2;
```

Comment:
Do we still want to prioritize `isInstalled` wallet in desktop though?

Reply:
Updated

The updated UI should show only one installed method instead of all installed methods

Reply:
So normally Keplr has priority?

Reply:
Yes, same as the order of the array items

Keplr > Cosmostation > Leap